4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4120 @@
+{
```

Comment:
<p>Maybe add a plot that shows prior likelihood and posterior together to show how prior and likelihood combine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Done</p>

Reply:
<p>Looks great!</p>

Reply:
<p>Is there a way to plot likelihood too?</p>

Reply:
<p>Agreed</p>